src/main/java/de/retest/recheck/review/ignore/io/ImportExternalFilterLoader.java

```diff
@@ -27,7 +27,7 @@ public ImportExternalFilterLoader() {
 			final Filter loaded = SearchFilterFiles.getFilterByName( reference );
 			return Optional.of( new ImportedExternalFilter( reference, loaded ) );
 		} catch ( final Exception e ) {
-			log.error( "Exception loading referenced filter '{}'.", reference );
+			log.error( "Exception loading referenced filter '{}': {}", reference, e );
```

Comment:
Exceptions do not need a parameter. It suffices to do the following:
```suggestion
			log.error( "Exception loading referenced filter '{}'.", reference, e );
```
Otherwise, use only the message
```suggestion
			log.error( "Exception loading referenced filter '{}': {}.", reference, e.getMessage() );
```

Reply:
Hmm. I tried both. Problem is that your first suggestion logs the complete stack trace (much too verbose), while the second option ... well unfortunately, `FileNotFoundException` doesn't have a message...

Reply:
And what does your implementation output?